github/types/ghtypes.go

```diff
@@ -0,0 +1,263 @@
+package types
+
+import (
+	"context"
+	"errors"
+	"strconv"
+
+	"github.com/athenianco/cloud-common/report"
+)
+
+var ErrNoAppMeta = errors.New("no application metadata")
+
+// AthenianAccID is a Athenian Github Account ID.
+// It combines a Github AppID with a Github InstallID and can uniquely identify an installation.
+type AthenianAccID int64
```

Comment:
Let's take this chance to rename it to `AccID`.

Reply:
Done.